package.json

```diff
@@ -67,10 +67,10 @@
     "simple-git": "1.65.0"
   },
   "scripts": {
-    "build": "babel src --out-dir dist && echo '\n\\033[0;34m[+] \\033[0;32mBuild done\\033[0m'",
-    "build:watch": "onchange 'src/**/*.js' 'node_modules/forest-express/dist/*' --no-exclude -i -- yarn build",
+    "build": "babel src --out-dir dist",
```

Comment:
Same question as `forest-express`: does the `echo` prevent you from doing something with the intent of helping to debug while implementing features.

Reply:
replied in forest-express